apps/web/src/app/(pages)/(dashboard)/roles/page.tsx

```diff
@@ -33,10 +53,57 @@ export default function Roles({
     reviews.data ? reviews.data[0] : undefined,
   );
 
+  const [showAlert, setShowAlert] = useState(true);
+
   return (
     <>
       <SearchFilter />
       {/* TODO: Loading animations */}
+      {error && showAlert && (
+        <div
+          id="bad-input"
```

Comment:
i think we should make this a component so we can reuse this toast pop up in other places, @RishikeshNK thoughts?

Reply:
Yup! I agree. We can move this to a separate component that takes in the `message` as a prop.